### ISSUE TITLE
Survey

### ISSUE THREAD

**Issue by samwilson** [geogeeks-au/geogeeks-au.github.io#13]
Questions:
* Night of week
* Which of these are you interested in?
  * Workshop (free text: what sort?)
  * Speakers
  * Mapping Party (ground truthing)
  * Mapathon (editing)
  * Lightning talks/open mic night
  * Hack nights/hackathons
* Where?
* Barriers to attending?
* Would online be benefitial?
* Can you help organise talks, workshops, or places to map?

Schedule:
* Jan: Speakers, 2 or three
* Feb: Mapathon, intro level
* Mar: Open mic night
* Apr: Mapping party followed by a 2nd mapathon
* May: speaker

**Comment by johnwbryant** [comment]
Draft survey here: https://docs.google.com/forms/d/e/1FAIpQLScWbfk4qulWsf1nduadXSXQWK4qKPP0TQRoWw08jRtxP4FqGQ/viewform

**Comment by samwilson** [comment]
Looks good to me!

Only tiny thing: should it say "Spacecubed (Riff)" or something, in case people know it only by the new name?

**Comment by johnwbryant** [comment]
Good point! Updated.

**Comment by samwilson** [comment]
I assume we should wait till after the survey to ask for bookings at Spacecubed shouldn't we?

**Comment by keithamoss** [comment]
Nice job! :)

I've sprinkled a little unicorn dust:

> Following the successful Perth hub of FOSS4G SotM Oceania a few weeks ago, we’re planning a fun year of events for the Perth open geospatial community. We heard from many of you that you wanted more chances to catch-up, chat, and learn with the rest of our community.
>
> So we've been thinking. We’re thinking guest speakers, OpenStreetMap mapathons and mapping parties, technical workshops, and more, maybe monthly. Possibly continuing at Spacecubed (Riff) in the CBD, or roaming around a bit.
>
> We’d love to get your input! If you can give us 3 to 5 minutes to fill out this survey, you’ll be helping us to craft a great series of events and bring together the community. Please share with your networks, the more the merrier.
>
> And we can always use more help! If you’re interested in getting involved, please [join our Slack group](https://docs.google.com/forms/d/e/1FAIpQLSc9j_tF-cKfldBRFLSN5ToiVYPpbFgPXeWUw55b5mdE1zRDDg/viewform?urp=gmail_link&gxids=7628), or reply to this email.
>
> Looking forward to seeing you in early 2021!

**Comment by keithamoss** [comment]
1. The logo in the header was pretty low-res. I've replaced it with the high-res PNG version that's [here](https://github.com/geogeeks-au/geogeeks-au.github.io/blob/master/img/geogeeks_logo.png).
2. I've had a go at adding hastily cobbled together descriptions of some of the events/activities for folks who might not know what they are. If someone could review and polish them I'd be most appreciative :)

**Comment by johnwbryant** [comment]
Thanks @keithamoss! Nice touches. I've updated and will send out one more test now. I'll aim to send out this afternoon, still happy to take suggestions & edits meanwhile.

>@samwilson: I assume we should wait till after the survey to ask for bookings at Spacecubed shouldn't we?

I think we need to do some reckoning on dates. My guess is that T-W-Th will all be OK, but we should get a better sense once we get a few responses from the survey.

Just taking a quick look at the last Thursday of each month, it doesn't seem to conflict with any holidays, which is a good start. I've added a tentative item to the Geogeeks public calendar here: https://calendar.google.com/calendar/u/4?cid=Z2VvZ2Vla3MucGVydGhAZ21haWwuY29t

Maybe we should give the survey a week or so, and regroup next week on Slack to lock in a schedule so we can book?
